### PR TITLE
Staticcheck:test/integration/examples

### PR DESCRIPTION
**Pull request by xiaoanyunfei**
**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:
Fix static check failures for
>test/integration/examples

**Which issue(s) this PR fixes**:

Ref: #https://github.com/kubernetes/kubernetes/issues/81657

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: test/integration/examples/apiserver_test.go [hunk 1]
```diff
@@ -131,7 +131,7 @@ func TestAggregatedAPIServer(t *testing.T) {
 
 	// wait for the unavailable API service to be processed with updated status
 	err = wait.Poll(100*time.Millisecond, 5*time.Second, func() (done bool, err error) {
-		_, err = kubeClient.Discovery().ServerResources()
+		_, _, err = kubeClient.Discovery().ServerGroupsAndResources()
```
Comment (BenTheElder) [inline test/integration/examples/apiserver_test.go]: which this change?

**Comment by BenTheElder** [comment]
can you explain why these changes were made other than "so the lints pass" ?

File: test/integration/examples/apiserver_test.go (hunk 1 above)
Comment (xiaoanyunfei) [reply test/integration/examples/apiserver_test.go:134]: > which this change?

`ServerResources` is marked Deprecated

**Pull request closed without merging**